configure

```diff
@@ -1500,6 +1542,12 @@ case "${ARCH}" in
             CFLAGS="${CFLAGS} -DX86_FEATURES"
             SFLAGS="${SFLAGS} -DX86_FEATURES"
 
+            check_xsave_intrinsics
```

Comment:
This should definitely go in alphabetical order, at the bottom of this if statement.

Reply:
I added it here, because it's only relevant to `x86_features.h`. But I can move it...